skins/oasis/js/ArticleLengthAbTesting.js

```diff
@@ -0,0 +1,28 @@
+require(['jquery', 'wikia.window', 'wikia.tracker'], function($, win, tracker) {	
+	'use strict';
+
+	var WIDTH_SAMPLING_RATIO = 1,
+		ARTICLE_LENGTH_THRESHOLD = 800,
+		SCALE = 100;
+		
+	if (Math.random() * 100 < WIDTH_SAMPLING_RATIO && win.wgIsArticle)  {
+		var labelPrefix,
```

Comment:
https://github.com/Wikia/guidelines/blob/master/JavaScript/CodingConventions.md#variables
Rule is a rule :) Don't assign values, just declare var names.